wp-content/themes/missilethreat/template-parts/sidebar.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * Sidebar for archive pages
+ *
+ * @package CSIS iLab
+ * @subpackage @package MissileThreat
+ * @since 1.0.0
+ */
+
+function get_archive_post_type() {
+	return is_archive() ? get_queried_object()->name : false;
+}
+
+if( get_archive_post_type() ) {
+	$query = new WP_Query( array(
+			'post_type' => get_archive_post_type(),
```

Comment:
Right now, you're running the `get_archive_post_type()` function 3 times. If you set it equal to a variable, you can run it once. Eg. `$post_type = get_archive_post_type()` and then use it on this line (`'post_type' => $post_type`) and then in the if/else statement down below to change the archive type text.